tenets/codelingo/code-review-comments/FULL_TEXT.md

```diff
@@ -472,6 +472,8 @@ Clarity of docs is always more important than saving a line or two in your funct
 Finally, in some cases you need to name a result parameter in order to change
 it in a deferred closure. That is always OK.
 
+[Tenet](https://github.com/codelingo/codelingo/tree/master/tenets/codelingo/code-review-comments/named-result-params)
```

Comment:
Is this tenet supposed to address the whole paragraph? It seems limited to a single case where type and name are equal. If it's limited, we should move the annotation somewhere sensible. Otherwise we need to state exceptions or TODOs for exceptions.

Reply:
Moved